### DISCUSSION TITLE
Release 1.1.0-beta

### DISCUSSION THREAD

**Discussion by monacofj** [fossguild/sucury#112] [General]
### Discussion about what features should be present in the upcoming release.

Milestone [Release 1.1.0-beta](https://github.com/courselab/sucury/milestone/1).

**Sucury 1.1.0 release plan**

- ~~Prerelease Sucury 1.1.0-beta~~ ~~2023-10-31 20:00 GMT-3~~  ~~Nov 2023-14-14 20:00 GMT-3~~
- ~~Release of Sucury 1.0.0~~  ~~Nov 2023-11-14 20:00 GMT-3~~ ~~Nov 2023-11-28 20:00 GMT-3~~

- Release of Sucury 1.0.0  2023-12-20 11:00 GMT-3   (Dec 21 by the morning)   **HARD DEADLINE**

**ToDo List**

~~- Decide what should go into 1.1.0-beta (discussion page) until 2023-10-24 20:00 GMT-3~~
~~- Link relevant issues to the Release 1.0.0-beta [milestone](https://github.com/courselab/sucury/milestone/1).~~
~~- Freeze 1.1.0-beta at 2023-10-24 20:00 GMT-3 (release branch)~~
~~-Ensure `AUTHORS`, `Changelog`, `NEWS,` and `docs\* `are updated before the beta release date.~~
~~-Work around the clock so as not to miss the release date.~~
~~-Work in the `develop` branch will proceed normally, without interruption.~~

See ```sucury/TODO```

**Comment by brun0-matheus** [comment]
I've added issues related to bugs to the milestone, except issues #91 (this shouldn't be a bug) and #82.
